README.md

```diff
@@ -1394,6 +1395,12 @@ Specifying the backend URL, needed for US hosted backend infrastructure.
 yarn tr-request-export --auth=$TRANSCEND_API_KEY --transcendUrl=https://api.us.transcend.io
 ```
 
+With Sombra authentication
+
+```sh
```

Comment:
i think tr-request-restart also needs an update. its only needs `--sombraAuth` if also the `--copyIdentifiers` flag is provided